### DISCUSSION TITLE
What framework is used to write the blog?

### DISCUSSION THREAD

**Discussion by RedDocMD** [phil-opp/blog_os#1041] [Questions] [answered]
From the `blog/contents` folder I can see that most of the blog is written in Markdown (except the diagrams, of course). So I was wondering what "framework" is being used to make a website out of it?

**Comment by bjorn3** [comment, marked as answer]
Zola: https://github.com/phil-opp/blog_os/blob/27ab4518acbb132e327ed4f4f0508393e9d4d684/.github/workflows/build-site.yml#L32

See https://www.getzola.org/
